libraries/GCS_MAVLink/GCS.h

```diff
@@ -1236,6 +1236,7 @@ class GCS
 
     enum class Option {
       GCS_SYSID_ENFORCE = (1U << 0),
+      IGNORE_PARAM_SET  = (1U << 1),
```

Comment:
I guess there's a parameter description update to go with this..

Reply:
If it is persistent don't you have to change the default for SCR_ENABLE to true? otherwise how do you set SCR_ENABLE = 1 on a new install?

Reply:
> If it is persistent don't you have to change the default for SCR_ENABLE to true? otherwise how do you set SCR_ENABLE = 1 on a new install?

Well, there's lots of ways to get that effect when building for yourself... and also, you can set that state before installing the script :-)

I'm changing this to be ephemeral state on the next version of this PR, so no longer relevant problem anyway :-)